app/server.go

```diff
@@ -355,24 +688,7 @@ func (a *App) StartServer() error {
 }
 
 func (a *App) StopServer() {
-	if a.Srv.Server != nil {
-		ctx, cancel := context.WithTimeout(context.Background(), TIME_TO_WAIT_FOR_CONNECTIONS_TO_CLOSE_ON_SERVER_SHUTDOWN)
-		defer cancel()
-		didShutdown := false
-		for a.Srv.didFinishListen != nil && !didShutdown {
-			if err := a.Srv.Server.Shutdown(ctx); err != nil {
-				mlog.Warn(err.Error())
-			}
-			timer := time.NewTimer(time.Millisecond * 50)
-			select {
-			case <-a.Srv.didFinishListen:
-				didShutdown = true
-			case <-timer.C:
-			}
-			timer.Stop()
-		}
-		a.Srv.Server.Close()
-		a.Srv.Server = nil
+	if a.Srv != nil && a.Srv.Server != nil {
```

Comment:
This looks like no needed code, or something to do.

Reply:
Yup. Should be removed!